editor/scene_tree_dock.cpp

```diff
@@ -1087,6 +1089,11 @@ void SceneTreeDock::_tool_selected(int p_tool, bool p_confirm_override) {
 					if (node->is_unique_name_in_owner()) {
 						continue;
 					}
+					// Root Node should never be unique name in its own scene!
+					if (first_selected->get() == EditorNode::get_singleton()->get_edited_scene()) {
+						editor_selection->remove_node(first_selected->get());
+						continue;
+					}
```

Comment:
This doesn't seem to work correctly. The root node is removed from selection, but the unique names are not toggled.

Reply:
My sincerest apologies for having you recompile this code (and more in the past few days) just to make my rushed patches worked in the first place.